cluster/images/conformance/conformance-e2e.yaml

```diff
@@ -59,7 +59,13 @@ spec:
     imagePullPolicy: IfNotPresent
     env:
     - name: E2E_FOCUS
-      value: "Pods should be submitted and removed"
+      value: "[Conformance]"
+    - name: E2E_SKIP
+      value: ""
```

Comment:
So we deliberately dont skip any tests when running conformance tests in CI?

Sonobuoy currently defaults to skip = `'Alpha|\[(Disruptive|Feature:[^\]]+|Flaky)\]'` but I've not really known where that comes from historically. Off the top of my head I"m not sure how many (if any) fall into that category. I think some feature tests do though.

Reply:
we talked about this in https://github.com/kubernetes/test-infra/pull/9104 but i mis-remembered the outcome. will switch up to use the skip list from there:
https://github.com/heptio/sonobuoy/blob/master/pkg/client/mode.go#L42

Reply:
@timothysc @spiffxp ^^